src/main/kotlin/com/hedvig/lookupservice/config/PostalConfig.kt

```diff
@@ -29,8 +29,17 @@ class PostalConfig {
 
     @Autowired
     fun populatePostalCodes() {
-        if (shouldPopulate(CountryCode.SE)) {
-            populatePostalCodesOfCountry(CountryCode.SE)
+        val shouldPopulate = environment.activeProfiles.contains("PopulatePostalCodes")
+        if (!shouldPopulate) {
+            return
+        }
+        val folder = File(postalCodesLocation)
+        val files = folder.listFiles()!!
+        val countryCodes = files
+            .map { file -> file.name.split(".txt")[0] }
```

Comment:
In my oppinion I think this is clearer :slightly_smiling_face: 
```suggestion
            .map { file -> file.name.replace(".txt", "") }
```